boto3/s3/inject.py

```diff
@@ -91,6 +94,27 @@ def download_file(self, Bucket, Key, Filename, ExtraArgs=None,
         extra_args=ExtraArgs, callback=Callback)
 
 
+def copy_file(self, SrcBucket, SrcKey, DestBucket, DestKey,
+              ExtraArgs=None, Callback=None, Config=None):
+    """Copy one S3 object to another s3 object.
+
+    Usage::
+
+        import boto3
+        s3 = boto3.resource('s3')
+        s3.meta.client.copy_file('bucket1', 'hello.txt', 'bucket2', 'hello.txt')
```

Comment:
Furthermore, I need to copy between accounts. So credentials for accessing bucket1 (source files) ARE NOT the same as credentials for bucket2 (dest files)...